### DISCUSSION TITLE
Mojolicious in a SELINUX MLS environment

### DISCUSSION THREAD

**Discussion by fskale** [mojolicious/mojo#1649] [General]
Hi,
i'm deploying a **Mojolicious/Minion App** in a high security environment.
I really didn't find any hint about **SELINUX MLS** and Mojolicious.
So about a year ago, i started to take a deep dive into **SELINUX**.
I can provide some anwsers/hints howto to start a project using **CENTOS7/8 (Redhat) and SELINUX MLS** (users, policies, labels, path considerations etc.)

Best regards
Franz

**Comment by s1037989** [comment]
I'm also working in a high security MLS environment and am interested in
deploying Mojolicious and especially Minion. I would love to know more
about what you have learned in order to be able to make that happen!
